data/util.py

```diff
@@ -773,3 +751,18 @@ def from_str(cls, value) -> Self:
     @classmethod
     def names(cls) -> list[str]:
         return [member.value for member in iter(cls)]
+
+
+class NeedsUpload(Enum):
+    YES = 0
+    NO_CHANGE = 1
+    RECENT_CHANGE = 2
+
+    def __bool__(self):
+        match self:
+            case NeedsUpload.YES:
+                return True
+            case NeedsUpload.NO_CHANGE | NeedsUpload.RECENT_CHANGE:
+                return False
+            case _:
```

Comment:
I'm wondering if this default case is even possible... What could be of type `NeedsUpload` but not part of the acceptable values for it? Unless the class is instantiated manually but that's not what you are supposed to do with `Enum` I think.
Just saying, because if we get rid of the default case, the method could be simplified to `return NeedsUpload.YES == self`.

Reply:
A future enum member?

It's supposed to be deleted anyway once we get rid of papi as a base, but I agree with you on that.

I'm working in the continuity of this branch, so don't hesitate to repeat your review on the future PR I'll open